src/test/resources/features/shoppingCart.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Tests for shopping cart
```

Comment:
Please use tags for the scenario/feature

Reply:
Used